scenarios/kubernetes_kops_aws.py

```diff
@@ -124,11 +124,11 @@ def main(args):
         if not os.path.basename(workspace) == 'kops':
             raise ValueError(workspace)
         version = 'pull-' + check_output('git', 'describe', '--always').strip()
-        check('make', 'gcs-publish-ci', 'VERSION=%s' % version)
-        gcs = 'gs://kops-ci/pulls/pull-kops-e2e-kubernetes-aws-scenario'
+        gcs = 'gs://kops-ci/pulls/%s' % os.getenv('JOB_NAME', 'pull-kops-e2e-kubernetes-aws')
```

Comment:
if `JOB_NAME` is not set, it will use `pull-kops-e2e-kubernetes-aws` as the default.